threads.ipynb

```diff
@@ -988,6 +988,9 @@
     "           To facilitate your work, we pass the size of the file (in number of characters) using the <code>size</code> argument.\n",
     "        </li>\n",
     "        <li>\n",
+    "            The input <code>n_processes</code> determines how many processes your solution should use.\n",
+    "        </li>\n",
+    "        <li>\n",
```

Comment:
Add a note that says "In case the test fails with BrokenProcessPool` consider moving the definition of `count_words` in a separate file and importing it in the notebook"

Reply:
Fixed in  c07c5f